distribution/lib/Standard/Table/0.0.0-dev/src/Delimited/Delimited_Format.enso

```diff
@@ -190,6 +189,6 @@ Delimited_Format.from (that : JS_Object) =
     unsupported_fields = ["value_formatter", "line_endings", "comment_character"]
     case unsupported_fields.find that.contains_key if_missing=Nothing of
         Nothing ->
-            Delimited_Format.Delimited delimiter=delimiter encoding=encoding headers=headers skip_rows=skip_rows row_limit=row_limit quote_style=quote_style keep_invalid_rows=keep_invalid_rows
+            Delimited_Format.Delimited delimiter=delimiter encoding=encoding headers=(Headers.from headers) skip_rows=skip_rows row_limit=row_limit quote_style=quote_style keep_invalid_rows=keep_invalid_rows
```

Comment:
It feels like this change should be unnecessary: the `:Headers` type annotation on the constructor should trigger the conversion automatically.

Of course there is no harm in keeping it either.